Dockerfile.test

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17-alpine
+FROM golang:1.19-alpine
```

Comment:
Why this change?

Reply:
following the 'boringcrypto' fix, we were required to upgrade things to 1.19